src/bp/ui-studio/src/web/views/Config/index.tsx

```diff
@@ -440,7 +450,7 @@ class ConfigView extends Component<Props, State> {
                 intent="primary"
                 icon="floppy-disk"
                 disabled={this.state.isSaving}
-                onClick={this.saveChanges}
+                onClick={() => this.saveChanges()}
```

Comment:
```suggestion
                onClick={this.saveChanges}
```

Reply:
This was changed on purpose, so "event" is not set when clicking the button. 
`this.saveChanges` is used by the button & the shortcut key handler, and we want to preventDefault when pressing ctrl+s, but you don't want to prevent the button click

Reply:
Then you should move the `event.preventDefault()` into the keyHandlers and remove `event` from `saveChanges(event?)`

This logic is too easy to miss if it's hiddent inside `saveChanges(event)`.

Reply:
Good point, guess I went the lazy route haha